sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -139,9 +139,10 @@ public virtual RecognizeContentOperation StartRecognizeContent(Stream form, Reco
                 FormContentType formContentType = recognizeContentOptions.ContentType ?? DetectContentType(form, nameof(form));
 
                 Response response = ServiceClient.AnalyzeLayoutAsync(
-                    formContentType.ConvertToContentType1(),
-                    recognizeContentOptions.Language,
+                    formContentType,
                     recognizeContentOptions.Pages.Count == 0 ? null : recognizeContentOptions.Pages,
+                    recognizeContentOptions.Language,
+                    null, // TODO: pass reading order.
                     form,
                     cancellationToken);
```

Comment:
For some reason they changed the order of the parameters in the swagger, which changed their order in the internal REST client as well.

Reply:
Passing null to the new parameters for now, but work to be done already being tracked in https://github.com/Azure/azure-sdk-for-net/issues/18805.

Reply:
>For some reason they changed the order of the parameters in the swagger

TA does that sometimes too :(